src/useCustomCompareEffect/__docs__/story.mdx

```diff
@@ -40,13 +40,12 @@ export function useCustomCompareEffect<
 
 #### Arguments
 
-- **callback** _`EffectCallback`_ - Function that will be passed to underlying effect hook.
-- **deps** _`React.DependencyList`_ - Dependencies list like for `useEffect`. If not undefined -
-  effect will be triggered when deps change AND conditions satisfy predicate.
-- **comparator** _`DependenciesComparator<Deps> `_ - Function that compares two dependencies
-  arrays, and returns true in case they're equal.
-- **effectHook** _`EffectHook<Callback, Deps, HookRestArgs>`_ - Effect hook that will be used to
-  run callback. Must comply `useEffect` signature, meaning that callback should be placed as first
-  argument and dependencies list as second.
-- **...effectHookRestArgs** _`HookRestArgs`_ - Extra arguments that passed to effectHook. Meaning
-  the arguments that are after 2nd argument.
+- **callback** _`EffectCallback`_ -  Function that will be passed to the underlying effect hook.
+- **deps** _`DependencyList`_ - Dependency list like the one passed to `useEffect`.
```

Comment:
Is it okay to drop the `If not undefined - effect will be triggered when deps change AND conditions satisfy predicate.` bit?

Reply:
I deleted it out of routine. In this case I should probably add it back since it is worth highlighting here that merely the dependencies chaning will not trigger the effect.

Reply:
I think that part is just copy-pasta from `useConditionalEffect`. There is no predicates to be satisfied in `useCustomCompareEffect`. There is the custom comparator which that bit may be referring to, but I don't think that bit is necessary anyway.